src/Mobs/PathFinder.cpp

```diff
@@ -0,0 +1,233 @@
+
+
+#include "Globals.h"
+#include "PathFinder.h"
+#include "../Chunk.h"
+
+cPathFinder::cPathFinder(double a_MobWidth, double a_MobHeight)
+	: m_Path(nullptr)
+	, m_GiveUpCounter(0)
+	, m_TicksSinceLastPathReset(1000)
```

Comment:
Would you mind explaining how exactly that'd be done? I find counting ticks essential:
 - It limits the frequency of how many times a new cPath is created, which is good for performance.
 - If recalc is based solely on distance between the final location and the destination last given to the pathfinder (those two were once called `FinalDestination` and `PathFinderDestination`), recalculation may not happen when it should. (I got an example if you'd like)